binder/postBuild

```diff
@@ -1,2 +1,5 @@
 pip install -e .
-jlpm build
\ No newline at end of file
+jlpm build
+
+# This seems to be explicitly needed with `pip install -e .`
+jupyter serverextension enable jupyterlab --sys-prefix
```

Comment:
I've run into this with serverextensions before as well. It seems the `conf.d` stuff doesn't get linked right with an editable install.